migrations/2017_09_17_000000_create_roles_table.php

```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-class CreateRolesTable extends Migration
+return new class extends Migration
```

Comment:
We can't make this change and keep `composer.json` allowing Laravel versions before 8, as support for anonymous migrations was introduced in Laravel 8

see: https://github.com/laravel/framework/pull/36906

Reply:
Removing previous Laravel versions would require a new major version